### DISCUSSION TITLE
Does avalonia support get touch size?

### DISCUSSION THREAD

**Discussion by AtomKing** [AvaloniaUI/Avalonia#10699] [Q&A] [answered]
my input device is a touch screen ,i can get touch point ,but how to get this point's size?
The data reported by my touch screen contains touch area data,
in wpf,i can get this parameter by TouchEventArgs.GetTouchPoint(xxx).Bounds

**Comment by maxkatz6** [comment, marked as answer]
No, there is no API for that yet. But in 11.1 we can have https://learn.microsoft.com/en-us/uwp/api/windows.ui.input.pointerpointproperties.contactrect?view=winrt-22621 ported, if somebody want to help with this.

**Reply by AtomKing** [reply]
thanks for your replay,maybe i can use other api to get touch data

**Reply by timunie** [reply]
@AtomKing or, if you have time to do, you can try to add the function and create a PR for it. But please wait for 11.0 release as we are full of work atm.
